### DISCUSSION TITLE
Prevention of turning on the Hires fix toggle in the txt2img section while using any accelerated model

### DISCUSSION THREAD

**Discussion by cleverestx** [VoltaML/voltaML-fast-stable-diffusion#77] [Ideas]
![image](https://github.com/VoltaML/voltaML-fast-stable-diffusion/assets/28905253/bc4fe1e9-09e8-4885-9e94-e8a666da053c)

At the very least text here saying "If using accelerated model, Hi-res feature will not work", but would be better to drop lock/grey out this toggle if an accelerated model is loaded in the UI

**Comment by Stax124** [comment]
Added in 5212b79
